kani-driver/src/call_single_file.rs

```diff
@@ -87,16 +87,17 @@ impl KaniSession {
             flags.push("--ignore-global-asm".into());
         }
 
-        // When running regression runs, the write_goto_binary option is enabled
-        // by defining an environment variable.
-        // The value must be non-empty and different from zero.
-        let enable_write_goto_binary = match std::env::var_os("KANI_ENABLE_WRITE_GOTO_BINARY") {
-            Some(str) => !str.is_empty() && str != "0",
-            None => false,
-        };
+        // // When running regression runs, the write_json_symtab option is enabled
```

Comment:
As discussed offline, perhaps we can turn this into a compile-team feature like `unsound_experiments` so that we can get rid of the environment variable. @celinval how does this sound?

Reply:
Remind me why we are keeping the JSON version?

Reply:
We keep this as a fallback option to help users workaround cases where goto binary would not behave as expected. We will deprecate JSON export later down the road.